app_gui

```diff
@@ -0,0 +1,406 @@
+
+class UniversityManagementApp(QWidget):
+    def __init__(self):
+        super().__init__()
+        self.initUI()
+        
+    def initUI(self):
+        self.setGeometry(100, 100, 400, 300)
+        self.setWindowTitle('University Management Application')
+        
+        layout = QVBoxLayout()
+
+        # Welcome Label
+        self.welcome_label = QLabel("Hello, Welcome to University Management Application", self)
+        layout.addWidget(self.welcome_label)
+        
+        # ComboBox for Management Selection
+        self.combo_box = QComboBox(self)
+        self.combo_box.addItem("Select Management")
+        self.combo_box.addItem("Student Management")
+        self.combo_box.addItem("Worker Management")
+        layout.addWidget(self.combo_box)
+        
+        # Button to Manage
+        self.manage_button = QPushButton("Manage", self)
+        self.manage_button.clicked.connect(self.manage)
+        layout.addWidget(self.manage_button)
+        
+        self.setLayout(layout)
+    
+    def manage(self):
+        choice = self.combo_box.currentIndex()
+        if choice == 1:
+            self.student_window = StudentManagement(self)
+            self.student_window.show()
+            self.hide()
+        elif choice == 2:
+            self.worker_window = WorkerManagement(self)
+            self.worker_window.show()
+            self.hide()
+        else:
+            QMessageBox.warning(self, "Error", "Please select a valid option")
+
+class StudentManagement(QWidget):
+    def __init__(self, parent):
+        super().__init__()
+        self.parent = parent
+        self.initUI()
+        
+    def initUI(self):
+        self.setGeometry(100, 100, 400, 300)
+        self.setWindowTitle('Student Management')
+        
+        layout = QVBoxLayout()
+
+        # Options for Student Management
+        self.option_label = QLabel("Select an option:", self)
+        layout.addWidget(self.option_label)
+        
+        self.option_combo = QComboBox(self)
+        self.option_combo.addItem("Select Option")
+        self.option_combo.addItem("Add Student")
+        self.option_combo.addItem("Find Student")
+        self.option_combo.addItem("Sort Students")
+        layout.addWidget(self.option_combo)
+        
+        self.option_button = QPushButton("Select", self)
+        self.option_button.clicked.connect(self.select_option)
+        layout.addWidget(self.option_button)
+        
+        self.setLayout(layout)
+    
+    def select_option(self):
+        choice = self.option_combo.currentIndex()
+        if choice == 1:
+            self.add_student_ui()
+        elif choice == 2:
+            self.find_student_ui()
+        elif choice == 3:
+            self.sort_students_ui()
+        else:
+            QMessageBox.warning(self, "Error", "Please select a valid option")
+
+    def add_student_ui(self):
+        self.add_student_window = QWidget()
+        self.add_student_window.setGeometry(100, 100, 400, 300)
```

Comment:
third time!!! same dimensions.
the idea is when you need to change the dimensions you will need to edit it three times but if they are global variables you will edit it once